spec/index.html

```diff
@@ -271,30 +272,6 @@ <h2>Terminology</h2>
       A string is a sequence of zero or more Unicode characters.</dd>
     <dt><code>true</code> and <code>false</code></dt><dd>
       Values that are used to express one of two possible boolean states.</dd>
-    <dt><dfn data-lt="internationalized resource identifier|iri|iris"><abbr title="Internationalized Resource Identifier">IRI</abbr></dfn></dt>
-    <dd>An <a>IRI</a> (Internationalized Resource Identifier) is a string that conforms to the syntax
-      defined in [[RFC3987]].</dd>
-    <dt><dfn data-lt="subject|subjects">subject</dfn></dt>
-    <dd>A <a data-cite="RDF11-CONCEPTS#dfn-subject">subject</a>
-      as specified by [[!RDF11-CONCEPTS]].</dd>
-    <dt><dfn data-lt="predicate|predicates">predicate</dfn></dt>
-    <dd>A <a data-cite="RDF11-CONCEPTS#dfn-predicate">predicate</a>
-      as specified by [[!RDF11-CONCEPTS]].</dd>
-    <dt><dfn data-lt="object|objects">object</dfn></dt>
-    <dd>An <a data-cite="RDF11-CONCEPTS#dfn-object">object</a>
-      as specified by [[!RDF11-CONCEPTS]].</dd>
-    <dt><dfn data-lt="rdf triple|rdf triples|triple|triples">RDF triple</dfn></dt>
-    <dd>A <a data-cite="RDF11-CONCEPTS#dfn-rdf-triple">triple</a>
-      as specified by [[!RDF11-CONCEPTS]].</dd>
-    <dt><dfn data-lt="rdf graph|rdf graphs|graph|graphs">RDF graph</dfn></dt>
-    <dd>An <a data-cite="RDF11-CONCEPTS#dfn-rdf-graph">RDF graph</a>
-      as specified by [[!RDF11-CONCEPTS]].</dd>
-    <dt><dfn data-lt="graph name|graph names">graph name</dfn></dt>
-    <dd>A <a data-cite="RDF11-CONCEPTS#dfn-graph-name">graph name</a>
-      as specified by [[!RDF11-CONCEPTS]].</dd>
-    <dt><dfn>default graph</dfn></dt>
-    <dd>The <a data-cite="RDF11-CONCEPTS#dfn-default-graph">default graph</a>
-      as specified by [[!RDF11-CONCEPTS]].</dd>
```

Comment:
Not sure we want to lose all these entries, although they definitely should not be exported terms and should use the `data-cite` mechanism. For someone not relying on on hyperlinks, having a basic definition of the terminology used in the spec in one place, whether imported or exported, is useful.

Reply:
except for IRI, none of these provide anything more than just links to their original definition, so I don't think removing them remove any local context; but naturally, I'm happy to reinstate them as data-cite if you prefer.

Reply:
Thanks, indeed, they provide only a local (possibly contextual) repetition of a term defined elsewhere. This comes down to a style point, more than anything. I think it best to keep them in place, and then later consider pruning back such terminology lists.

As this is out of your repo, it would be great if you'd put them back and use `<dfn data-cite="...">` so that they are just fancy references.

I'll create an issue about the need for local term definitions for imported terms.